apps/etherscan/src/app/utils/verify.ts

```diff
@@ -90,6 +90,8 @@ export const verify = async (
       const response = await axios.post(etherscanApi, body)
       const { message, result, status } = await response.data
 
+      console.log("response", response.data)
```

Comment:
remove log

Reply:
we need a script to detected added consoles  👯 😄